frontend/src/metabase/home/containers/HomepageApp.jsx

```diff
@@ -23,7 +20,6 @@ const mapStateToProps = (state, props) => ({
   activity: getActivity(state),
   recentViews: getRecentViews(state),
   user: getUser(state),
```

Comment:
I think this and the proptype can be deleted too.

Reply:
I tried, but it seems it is used by (passed down to) `Activity.jsx` component. I you remove it, there is a console error:
```
warning.js:33 Warning: Failed prop type: The prop `user` is marked as required in `Activity`, but its value is `undefined`.
    in Activity (created by HomepageApp)
    in HomepageApp (created by HomepageApp)
    in HomepageApp (created by RouterContext)
    in Unknown (created by UserIsAuthenticated(Component))
    in UserIsAuthenticated(Component) (created by Connect(UserIsAuthenticated(Component)))
    in Connect(UserIsAuthenticated(Component)) (created by MetabaseIsSetup(Connect(UserIsAuthenticated(Component))))
    in MetabaseIsSetup(Connect(UserIsAuthenticated(Component))) (created by Connect(MetabaseIsSetup(Connect(UserIsAuthenticated(Component)))))
    in Connect(MetabaseIsSetup(Connect(UserIsAuthenticated(Component)))) (created by RouterContext)
    in div (created by App)
    in ScrollToTop (created by ScrollToTop)
    in ScrollToTop (created by App)
    in App (created by App)
    in App (created by Title[App])
    in Title[App] (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in ThemeProvider
    in DragDropContextProvider
    in Provider
    ```

Reply:
Oops! I missed that.